redbot/cogs/economy/economy.py

```diff
@@ -904,16 +906,19 @@ async def slotmax(self, ctx: commands.Context, bid: positive_int):
         )
 
     @economyset.command()
-    async def slottime(self, ctx: commands.Context, seconds: int):
+    async def slottime(self, ctx: commands.Context, seconds: TimedeltaConverter):
```

Comment:
It probably makes sense to set default unit to seconds here:
```suggestion
    async def slottime(
        self, ctx: commands.Context, seconds: TimedeltaConverter(default_unit="seconds")
    ):
```